src/main/resources/assets/wotr/models/block/key_forge.json

```diff
@@ -1,306 +1,257 @@
 {
-	"parent": "block/block",
-	"credit": "Req",
+	"credit": "Made with Blockbench",
 	"texture_size": [64, 64],
 	"textures": {
-		"0": "wotr:block/key_forge",
-		"particle": "wotr:block/key_forge"
+		"0": "block/Key_Forge_V2_texture",
```

Comment:
These texture lines also need to be changed to match up with the actual texture identifiers.